java/src/main/java/com/training/db/hw2/Factorial.java

```diff
@@ -0,0 +1,40 @@
+package com.training.db.hw2;
+
+public class Factorial {
+
+    public static void calculateFactorial() {
+
+        int n2 = UiUtil.input("Input second number: ");
```

Comment:
call of this method can be moved to main function to avoid the code duplicating, and the argument could be passed as function parameters.

Reply:
done